syntax = "proto3";

package sea;

service SeaService {
  rpc SeaMethod(SeaRequest) returns (SeaResponse) {}
}

message SeaRequest { string request_sea_field = 1; }

message SeaResponse { string response_sea_field = 1; }